orga/tests/integration/components/participant/assessment/header_test.js

```diff
@@ -0,0 +1,203 @@
+          participantExternalId: null,
+          isShared: false,
+        };
+        this.campaign = {};
+
+        await render(hbs`<Participant::Assessment::Header @participation={{participation}} @campaign={{campaign}} />`);
+
+        assert.dom('li').exists({ count: 2 });
+      });
+    });
+  });
+
+  module('results information', function() {
+    module('when the participation is shared', function() {
+      test('it should not display campaign progression', async function(assert) {
+        this.participation = { progression: 100, isShared: true };
+        this.campaign = {};
+
+        await render(hbs`<Participant::Assessment::Header @participation={{participation}} @campaign={{campaign}} />`);
+
+        assert.notContains('Avancement');
+        assert.notContains('100%');
+      });
+
+      test('it displays campaign participation details of mastery percentage (validated skills over total skills)', async function(assert) {
+        this.participation = {
+          validatedSkillsCount: 45,
+          targetedSkillsCount: 50,
+          isShared: true,
+        };
+        this.campaign = {};
+
+        await render(hbs`<Participant::Assessment::Header @participation={{participation}} @campaign={{campaign}} />`);
+
+        assert.dom('[aria-label="Résultat"]').containsText('45 / 50 ACQUIS');
+      });
+
+      module('when the campaign has stages', function() {
+        test('it displays stages acquired', async function(assert) {
+          this.campaign = {
+            hasStages: true,
+            stages: [{ threshold: 20 }, { threshold: 70 }],
+          };
+          this.participation = {
+            masteryPercentage: 65,
+            isShared: true,
+          };
+
+          await render(hbs`<Participant::Assessment::Header @participation={{participation}} @campaign={{campaign}} />`);
+
+          assert.dom('[aria-label="Paliers"]').containsText('1 étoiles sur 2');
+        });
+      });
+
+      module('when the campaign has no stages', function() {
+        test('it displays campaign participation mastery percentage', async function(assert) {
+          this.participation = { masteryPercentage: 65, isShared: true };
+          this.campaign = {};
+
+          await render(hbs`<Participant::Assessment::Header @participation={{participation}} @campaign={{campaign}} />`);
+
+          assert.contains('65%');
+        });
+      });
+
+      module('when the campaign has badges', function() {
+        test('it displays badges acquired', async function(assert) {
+          this.campaign = { hasBadges: true };
+          this.participation = { isShared: true, badges: [{ id: 1, title: 'Les bases' }] };
+
+          await render(hbs`<Participant::Assessment::Header @participation={{participation}} @campaign={{campaign}} />`);
+
+          assert.dom('[aria-label="Résultats Thématiques"]').containsText('Les bases');
+        });
+      });
+
+      module('when the campaign has not badges', function() {
```

Comment:
**suggestion (non-blocking) :** `when the campaign has not badges` => no badges ?

Reply:
done